cypress/integration/embedded-mode.spec.ts

```diff
@@ -1,64 +1,51 @@
-const SOURCE_ID = 'c75ccdc4-418d-4ade-9059-90f0fc4ddbd1';
+const SOURCE_ID = 'fc126bd2-b903-4777-b7f4-c1c33410cc21';
```

Comment:
Is this checking a source in the viz itself? This edges towards this being an e2e test, I'd prefer it if we mocked this.

Reply:
what's the best way of mocking it?